colossalai/engine/schedule/_pipeline_schedule.py

```diff
@@ -830,4 +830,4 @@ def _backward_step_helper(microbatch_id):
             output, label = pack_return_tensors(return_tensors)
             return output, label, accum_loss
         else:
-            return None, None, accum_loss
+            return None, None, accum_loss
```

Comment:
Use precommit please.

Reply:
I have used pre-commit. but it still appeared.

Reply:
> Use precommit please.

Updated